android/src/main/java/com/tailscale/ipn/ui/notifier/Notifier.kt

```diff
@@ -4,19 +4,23 @@
 package com.tailscale.ipn.ui.notifier
 
 import android.util.Log
+import com.tailscale.ipn.ui.model.Ipn
 import com.tailscale.ipn.ui.model.Ipn.Notify
+import com.tailscale.ipn.ui.model.Netmap
 import kotlinx.coroutines.CompletableDeferred
 import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.job
 import kotlinx.coroutines.launch
+import kotlinx.serialization.ExperimentalSerializationApi
 import kotlinx.serialization.json.Json
-
-typealias NotifierCallback = (Notify) -> Unit
```

Comment:
No longer part of public API, so no need for type alias.

Reply:
Mainly habit... Complex closure types look terrible in method declarations, so I generally typealias them, but I'm inconsistent about it....